
---
std_msgs/Int16MultiArray bin_content_d
std_msgs/Int16MultiArray bin_content_e
std_msgs/Int16MultiArray bin_content_f
std_msgs/Int16MultiArray bin_content_g
std_msgs/Int16MultiArray bin_content_h
std_msgs/Int16MultiArray bin_content_i
std_msgs/Int16MultiArray bin_content_j
std_msgs/Int16MultiArray bin_content_k
std_msgs/Int16MultiArray bin_content_l
